spec/relativetimeformat.html

```diff
@@ -0,0 +1,403 @@
+<emu-clause id="relativetimeformat-objects">
+  <h1>RelativeTimeFormat Objects</h1>
+
+  <emu-clause id="sec-intl-relativetimeformat--abstracts">
+    <h1>Abstract Operations for RelativeTimeFormat Objects</h1>
+
+
+    <emu-clause id="sec-InitializeRelativeTimeFormat" aoid="InitializeRelativeTimeFormat">
+      <h1>InitializeRelativeTimeFormat ( _relativeTimeFormat_, _locales_, _options_ )</h1>
+
+      <p>
+        The abstract operation InitializeRelativeTimeFormat accepts the arguments _relativeTimeFormat_ (which must be an object), _locales_, and _options_. It initializes _relativeTimeFormat_ as a RelativeTimeFormat object.
+      </p>
+      <p>The following steps are taken:</p>
+
+      <emu-alg>
+        1. Let _requestedLocales_ be ? CanonicalizeLocaleList(_locales_).
+        1. If _options_ is *undefined*, then
+          1. Let _options_ be ObjectCreate(*null*).
+        1. Else,
+          1. Let _options_ be ? ToObject(_options_).
+        1. Let _opt_ be a new Record.
+        1. Let _matcher_ be ? GetOption(_options_, `"localeMatcher"`, `"string"`, «`"lookup"`, `"best fit"`», `"best fit"`).
+        1. Set _opt_.[[LocaleMatcher]] to _matcher_.
+        1. Let _numberingSystem_ be ? GetOption(_options_, `"numberingSystem"`, `"string"`, *undefined*, *undefined*).
+        1. If _numberingSystem_ is not *undefined*, then
+          1. If _numberingSystem_ does not match the `(3*8alphanum) *("-" (3*8alphanum))` sequence, throw a *RangeError* exception.
```

Comment:
Can we replace this line with the analogous line added in the [locale proposal](https://tc39.es/proposal-intl-locale/#sec-Intl.Locale): 

https://github.com/tc39/proposal-intl-locale/blob/895343957adf133bed9cb3c523a80371e9034d7c/spec.html#L209

I think its much more clear!